### ISSUE TITLE
Simple Linear Regression Nan's after a few iterations.

### ISSUE THREAD

**Issue by bcarpenter-pub** [fchollet/keras#3040]
I'm attempting to write a simple linear regression just to play around with Keras, however when training even a very simple net on simple data, the weights go to Nan.

```python
import numpy as np
import os
os.environ['KERAS_BACKEND'] = 'tensorflow'
import tensorflow as tf
from keras.models import Sequential
from keras.layers import Dense, Activation
from keras.optimizers import SGD

x = 1.0*np.array(range(200))
y = 2.0*x

model = Sequential()
model.add(Dense(1, input_dim=1))

sgd = SGD(lr=0.001, decay=1e-6, momentum=0, nesterov=False)
model.compile(loss='mean_squared_error', optimizer=sgd)
model.fit(x, y, nb_epoch=1, batch_size=1)
score = model.predict(x[:10], batch_size=10)
for layer in model.layers:
    weights = layer.get_weights()
    print layer
    print weights
print score
```

Am I doing something wrong?

**Comment by bcarpenter-pub** [comment]
realized my learning rate was too high.
